catapult-sdk/package.json

```diff
@@ -8,9 +8,9 @@
     "build": "ncp src/ _build && cross-env BABEL_ENV=production babel src -d _build_browser --source-maps",
     "rebuild": "npm run clean && npm run build",
     "test": "mocha --full-trace --recursive",
-    "test:coverage": "nyc yarn test && nyc report --reporter=text-lcov",
+    "test:coverage": "nyc npm test && nyc report --reporter=text-lcov",
```

Comment:
following @Vektrat earlier change, replacing `yarn` with `npm` in package.json files